Tests/iaas/flavor-naming/flavor-name-check.py

```diff
@@ -629,11 +629,14 @@ def readmandflavors(fnm):
                 break
     with open(fnm, "r", encoding="UTF-8)") as fobj:
         yamldict = yaml.safe_load(fobj)
-    ydict = yamldict["SCS-Spec"]["MandatoryFlavors"]
+    man_ydict = yamldict["SCS-Spec"]["MandatoryFlavors"]
```

Comment:
The `yamldict` is changed in place, so we could as well proceed as follows (modulo typos):

```
if prefer_old:
    for name_list in yamldict["SCS-Spec"].values():
        for i, name in enumerate(name_list):
            name_list[i] = new_to_old(name)
return yamldict["SCS-Spec"]["MandatoryFlavors"], yamldict["SCS-Spec"]["RecommendedFlavors"]
```

Then it's more obvious that the change is done in place, we get rid of the duplicated code, and the return statement is more telling (in my opinion).

Reply:
Implemented in https://github.com/SovereignCloudStack/standards/pull/294/commits/02044a8b183a92de04df03d30dfee12ab7fa4cb7